PIL/DdsImagePlugin.py

```diff
@@ -93,118 +92,16 @@
 DXT5_FOURCC = 0x35545844
 
 
-def _decode565(bits):
-    a = ((bits >> 11) & 0x1f) << 3
-    b = ((bits >> 5) & 0x3f) << 2
-    c = (bits & 0x1f) << 3
-    return a, b, c
-
-
-def _c2a(a, b):
-    return (2 * a + b) // 3
-
-
-def _c2b(a, b):
-    return (a + b) // 2
-
-
-def _c3(a, b):
-    return (2 * b + a) // 3
+def _dxt1(data, width, height):
+    return _dxtc.decode_dxt1(data, width, height)
```

Comment:
param data was stream, is now bytes; this breaks the DDS container code later in the file